### PR TITLE
Cassandra website is building/including the wrong versioned nodetool docs

### PR DESCRIPTION
**Pull request by michaelsembwever**
The fix is the first commit.

The second commit is regenerating the in-tree docs, with the fix. The second commit doesn't need to be reviewed, beyond seeing the fix in action.

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -7,36 +7,45 @@ export CASSANDRA_DIR="${BUILD_DIR}/cassandra"
...
+popd
 
-# Make sure we have the latest commit of Cassandra 3.11
+for v in $versions ; do
+  pushd ${CASSANDRA_DIR}
+  git clean -xdff
+  git checkout $v
+  popd
+  make .build-doc
+done
```
Comment (michaelsembwever) [inline docker-entrypoint.sh:32]: i can delete this now. (haven't generated docs for all the release tags in the following commit.)

Comment (michaelsembwever) [reply docker-entrypoint.sh]: done.

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -7,36 +7,44 @@ export CASSANDRA_DIR="${BUILD_DIR}/cassandra"
...
+popd
+make .build-doc .latest-doc-link
 
-# Now make the docs for the latest version
-cd ${CASSANDRA_SITE_DIR}/src
-make add-latest-doc
 
+# Uncomment for building docs for each of the released versions on the 3.11 branch
+#
+#versions="cassandra-3.11.5 cassandra-3.11.6 cassandra-3.11.7 cassandra-3.11.8"
```
Comment (anthony-grasso-datastax) [inline docker-entrypoint.sh:23]: Please add a comment saying that doc generations only work for version 3.11.5 and above.

**Review by anthony-grasso-datastax — COMMENTED** [review]
Nice find and fix @michaelsembwever ! A few minor comments, otherwise the fix looks good to me.

File: docker-entrypoint.sh [hunk 3]
```diff
@@ -7,36 +7,44 @@ export CASSANDRA_DIR="${BUILD_DIR}/cassandra"
...
+#
+#for v in $versions ; do
+#  pushd ${CASSANDRA_DIR}
+#  git clean -xdff
+#  git checkout $v
+#  popd
+#  make .build-doc
+#done
```
[the remark this answers is not in the document]
Comment (michaelsembwever) [reply docker-entrypoint.sh:31]: let's leave this to post-website-docs-rewrite. i'd like the build to regenerate it all, and not store anything in master. but let's see what the rewrite leave us with.

**Comment by michaelsembwever** [comment]
manually committed with https://github.com/apache/cassandra-website/commit/6ac702768d90421c4b8b1eb0762a4f3cca65bb1e

**Pull request closed without merging**